### PR TITLE
Split session statics reset method to prevent unloading seasonal backgrounds

### PR DESCRIPTION
**Pull request by dekrain**
Fixes #16180

### CODE REVIEW

File: osu.Game/Configuration/SessionStatics.cs [hunk 1]
```diff
@@ -24,6 +22,18 @@ public void ResetValues()
...
+        /// <summary>
+        /// Used to revert statics to their defaults after being idle for appropiate amount of time.
+        /// Does not affect loaded seasonal backgrounds.
+        /// </summary>
+        public void ResetValues()
+        {
+            ensureDefault(GetBindable<bool>(Static.LoginOverlayDisplayed));
+            ensureDefault(GetBindable<bool>(Static.MutedAudioNotificationShownOnce));
+            ensureDefault(GetBindable<bool>(Static.LowBatteryNotificationShownOnce));
+            ensureDefault(GetBindable<double?>(Static.LastHoverSoundPlaybackTime));
```
Comment (bdach) [inline osu.Game/Configuration/SessionStatics.cs:34]: Arguably `LastHoverSoundPlaybackTime` also shouldn't be reset. The inactivity period required is long enough to not matter here, but from a semantic point of view there's no reason to do anything with hover sound debounce logic if the user has been deemed inactive.

Interested in more thoughts on this in general from other reviewers.

Comment (peppy) [reply osu.Game/Configuration/SessionStatics.cs:34]: Can agree to this. Would probably also rename this method to be something like `ResetAfterInactivity`.

**Comment by peppy** [comment]
I've made some refactors to the naming and the structure of the code. Also fixed the related test not actually... working.. ever.

**Review by peppy — DISMISSED** [review]

File: osu.Game/Configuration/SessionStatics.cs (hunk 1 above)
Comment (bdach) [reply osu.Game/Configuration/SessionStatics.cs]: Removed the resetting of `LastHoverSoundPlaybackTime` myself in latest batch of commits.

**Comment by bdach** [comment]
I've piled on some of my own changes onto this branch, like removing an unused method, not resetting the `LastHoverSoundPlaybackTime` static and adding test coverage for the fail case here. Looks good otherwise.

**Review by bdach — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**